include/boost/decimal/detail/cmath/tanh.hpp

```diff
@@ -61,25 +61,39 @@ constexpr auto tanh(T x) noexcept -> std::enable_if_t<detail::is_decimal_floatin
             {
                 using coefficient_array_type = std::array<T, static_cast<std::size_t>(UINT8_C(11))>;
 
-                constexpr auto coefficient_table =
+                #if (defined(__clang__) && (__clang__ < 6))
+                #  pragma GCC diagnostic push
+                #  pragma GCC diagnostic ignored "-Wmissing-braces"
+                #endif
+
+                #if (defined(__clang__) && (__clang__ < 6))
+                const
+                #else
+                constexpr
+                #endif
+                auto coefficient_table =
```

Comment:
Would pre-computing the values fix clang < 6 issue?

Reply:
Yes. There is an expressional syntax that works for all the CI compielrs. I've got like 6 or seven tables and i modified tham all except for two. I'll eliminate these compiler switches and go `constexpr` on all of them prior to merging this.